doc/reference/controllers.md

```diff
@@ -463,31 +575,63 @@ Below table list other required fields for each supported DNS provider (Note: `g
+| randomizationFactor  | float64 | Randomization factor for actual wait duration, a number in interval `[0, 1]`, default is 0. The actual wait duration used is a random number in interval `[(base wait duration) * (1 - randomizationFactor),  (base wait duration) * (1 + randomizationFactor)]` | No |
 
-### mesh.MonitorMTLS
+#### circuitBreaker
 
-| Name         | Type                                 | Description                                     | Required |
-|--------------|--------------------------------------|-------------------------------------------------|----------|
-| enabled      | bool                                 | Enable monitor mTLS                             | Yes      |
-| url          | string                               | URL of monitor gateway to report metrics        | Yes      |
-| username     | string                               | Username for monitor gateway endpoint           | Yes      |
-| password     | string                               | Password for monitor gateway endpoint           | Yes      |
-| caCertBase64 | string                               | CA of PEM encoded data in base64 encoded format | Yes      |
-| certs        | [mesh.MonitorCert](#meshMonitorCert) | Certificate pairs of services                   | Yes      |
+The CircuitBreaker is a finite state machine with three states: `CLOSED`, `OPEN`, and `HALF_OPEN`. When the state is `CLOSED`, requests pass through the CircuitBreaker normally, state transits to `OPEN` if request failure rate or slow request rate reach a configured threshold and the CircuitBreaker short-circuiting all requests in this state. After a configured duration, state transits from `OPEN` to `HALF_OPEN`, in which a limited number of requests are permitted to pass through the CircuitBreaker while other requests are still short-circuited, and state transit to `CLOSED` or `OPEN` based on the results of the permitted requests.
 
-### mesh.MonitorCert
+When `CLOSED`, the CircuitBreaker uses a sliding window to store and aggregate the result of recent requests, the window can either be `COUNT_BASED` or `TIME_BASED`. The `COUNT_BASED` window aggregates the last N requests and the `TIME_BASED` window aggregates requests in the last N seconds, where N is the window size.
+
+Below is an example configuration with both `COUNT_BASED` and `TIME_BASED` policies. Policy `circuit-breaker-example-count` short-circuits requests if more than half of recent requests failed.
+
+Policy `circuit-breaker-example-time` short-circuits requests if more than 60% of recent requests failed.
+
+> failed means that backend filter returns non-empty results. 
+
+```yaml
+kind: CircuitBreaker
+name: circuit-breaker-example-count 
+slidingWindowType: COUNT_BASED
+failureRateThreshold: 50
+slidingWindowSize: 100
+
+---
+kind: CircuitBreaker
+name: circuit-breaker-example-time 
+slidingWindowType: TIME_BASED
+failureRateThreshold: 60
+slidingWindowSize: 100
+```
```

Comment:
please refer https://github.com/megaease/easegress/blob/663a08344cc64fe6194c3fbe454c912e518a5a3a/doc/cookbook/resilience.md#circuit-break to update this part, and please make sure the kind should be `CircuitBreak` instead of `CircuitBreaker`.